PhysicsTools/NanoAOD/python/electrons_cff.py

```diff
@@ -270,11 +263,12 @@ def _get_bitmapVIDForEle_docstring(modules,WorkingPoints):
     ecalTrkEnergyPostCorrNew    = cms.InputTag("calibratedPatElectrons94X","ecalTrkEnergyPostCorr"),
 )
 
-run2_nanoAOD_102Xv1.toModify(slimmedElectronsWithUserData.userFloats,
-    ecalTrkEnergyErrPostCorrNew = cms.InputTag("calibratedPatElectrons102X","ecalTrkEnergyErrPostCorr"),
-    ecalTrkEnergyPreCorrNew     = cms.InputTag("calibratedPatElectrons102X","ecalTrkEnergyPreCorr"),
-    ecalTrkEnergyPostCorrNew    = cms.InputTag("calibratedPatElectrons102X","ecalTrkEnergyPostCorr"),
-)
+for modifier in run2_nanoAOD_102Xv1,run2_nanoAOD_106Xv1:
+    modifier.toModify(slimmedElectronsWithUserData.userFloats,
+        ecalTrkEnergyErrPostCorrNew = cms.InputTag("calibratedPatElectrons102X","ecalTrkEnergyErrPostCorr"),
```

Comment:
@swertz 
Should this be calibratedPatElectrons102X for 102X and  calibratedPatElectrons106X for 106X ?